### DISCUSSION TITLE
need advice on how to do this

### DISCUSSION THREAD

**Discussion by JunkyardCat** [dbeaver/dbeaver#21614] [Q&A]
1. how do you copy column from one db to another? when i did the copy paste it didnt copy the name, type and other property
2. is there a way to create like a default column when you create a new table? example if i create tableA itll automatically create column 1... column X
3. is there a way to add rules on standardization or naming convention ? like linting for programming

**Comment by JunkyardCat** [comment]
anyone?

**Reply by LonwoLonwo** [reply]
Hello @JunkyardCat 

We do not have options for 2 and 3. 
ABout the first question... What database do you use?

**Reply by JunkyardCat** [reply]
sqlite, postgres
